src/main/resources/static/js/basic.js

```diff
@@ -332,8 +332,14 @@ var loading = {
 
 /* iFrame에서 호출 (보안문제로 broadcast하여 호출) */
 function receiveMessage(event) {
-	var data = JSON.parse(event.data);
-	
+	var data = event.data;
+
+	if (typeof data === 'object') {
+
+	} else if (typeof data === 'string') {
```

Comment:
Dear @ubermen5che ,
Is there any reason to leave the `if`? 
If the `if` statement is unnecessary, why not delete the `if` and changing `else if` to `if` ?